src/main/java/com/matera/cadastrocentral/authentication/AuthenticationService.java

```diff
@@ -0,0 +1,31 @@
+package com.matera.cadastrocentral.authentication;
+
+import com.matera.cadastrocentral.client.Client;
+import com.matera.cadastrocentral.client.ClientRepository;
+import com.matera.cadastrocentral.client.ClientService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class AuthenticationService {
+
+    private ClientService clientService;
+    private ClientRepository clientRepository;
+
+
+    @Autowired
+    public AuthenticationService(ClientService clientService, ClientRepository clientRepository) {
+        this.clientService =  clientService;
+        this.clientRepository = clientRepository;
+    }
+
+
+    public void changePassword(UUID clientId, PasswordDTO newPassword) {
+        Optional<Client> client =  clientService.getClientById(clientId);
+        client.get().setPassword(newPassword.getNewPassword());
```

Comment:
Maybe just validate before this line if the the newPassword is a valid password. Something like this: `if(StringUtils.isNumeric(clientDTO.getPassword()) && clientDTO.getPassword().length() == 6)`